.circleci/config.yml

```diff
@@ -0,0 +1,251 @@
+aliases:
+  - &restore-cache
+    keys:
+      - v1-dependencies-{{ .Branch }}-{{ checksum "package.json" }}
+      # Fallback in case checksum fails
+      - v1-dependencies-{{ .Branch }}-
+  - &save-cache
+    paths:
+      - node_modules
+    key: v1-dependencies-{{ .Branch }}-{{ checksum "package.json" }}
+
+  - &restore-cache-website
+    keys:
+      - v1-website-dependencies-{{ .Branch }}-{{ checksum "website/package.json" }}
+      # Fallback in case checksum fails
+      - v1-website-dependencies-{{ .Branch }}-
+  - &save-cache-website
+    paths:
+      - website/node_modules
+    key: v1-website-dependencies-{{ .Branch }}-{{ checksum "website/package.json" }}
+
+  - &restore-cache-danger
+    keys:
+      - v1-danger-dependencies-{{ .Branch }}-{{ checksum "danger/package.json" }}
+      # Fallback in case checksum fails
+      - v1-danger-dependencies-{{ .Branch }}-
+  - &save-cache-danger
+    paths:
+      - danger/node_modules
+    key: v1-danger-dependencies-{{ .Branch }}-{{ checksum "danger/package.json" }}
+
+  - &restore-cache-android
+    keys:
+      - v1-android-dependencies-{{ .Branch }}-{{ checksum "ReactAndroid/build.gradle" }}
+      # Fallback in case checksum fails
+      - v1-android-dependencies-{{ .Branch }}-
+  - &save-cache-android
+    paths:
+     - "ReactAndroid/build/downloads"
+    key: v1-android-dependencies-{{ .Branch }}-{{ checksum "ReactAndroid/build.gradle" }}
+
+  - &restore-cache-buck
+    keys:
+# TODO: Pick file to checksum against
+      - v1-buck-dependencies-{{ .Branch }}
+  - &save-cache-buck
+    paths:
+     - "~/buck"
+    key: v1-buck-dependencies-{{ .Branch }}
+
+version: 2
+jobs:
+  test-node-8:
+    working_directory: ~/react-native
+    docker:
+      - image: circleci/node:8
+    steps:
+      - checkout
+      - run: npm install --no-package-lock
+      - run: |
+          npm run lint
+          npm run flow -- check
+          npm test -- --maxWorkers=1
```

Comment:
CircleCI says there are 2 CPUs per build now, could try 2 workers (and for the other Node versions as well)